modules/stochastic_tools/test/tests/actions/parameter_study_action/gold/cartesian_product_csv_study_qoi_0001.csv

```diff
@@ -0,0 +1,26 @@
+average:value,study_qoi_transfer:converged
```

Comment:
Maybe a better name than `study_qoi_transfer` for users like `sample_study`?

Reply:
I can have it just be `converged` pretty easily, would that work?

Reply:
Yes. Or maybe `Success`?